scripts/run_imitation_example.sh

```diff
@@ -3,21 +3,11 @@
 set -e
 set -o pipefail
 
-# We need an older version of setuptools (and pip) to install gym 0.21. Poetry
-# doesn't support specifying the setuptools version used at build time.
-
-# The workaround is:
-
 # First check if we can already install the full environment without issues. In
 # that case we are done. This is the case if the environment was already
 # installed previously, i.e., this speeds up the process on the second run.
 if ! poetry install -E examples; then
```

Comment:
The `if` (and the comment) is no longer necessary. We just need the `poetry install -E examples` line.